### DISCUSSION TITLE
Comment field related to Column is not resizing. It is very small

### DISCUSSION THREAD

**Discussion by oc-tk** [dbeaver/dbeaver#14573] [General]
![Screenshot 2021-10-22 113839](https://user-images.githubusercontent.com/8511623/138431931-bcbec50e-f4b2-4efc-ae78-2cf4fd6b1fcc.png)

Very often we have a lot of text inside comment field and I cannot resize it manually. It would be nice to resize it manually or make it way bigger - there is a lof of wasted space.

**Comment by LonwoLonwo** [comment]
Hello @Spamm3r 

You can open resizable dialog from Table Viewer

![2021-10-22 13_39_44-DBeaver Ultimate 21 2 3 - language](https://user-images.githubusercontent.com/45152336/138440744-6739cdd6-d653-4184-88a5-360ce27bed1c.png)
